packages/react-components/src/components/styles/VerticalGallery.styles.ts

```diff
@@ -24,14 +24,18 @@ export const childrenContainerStyle: IStyle = {
 export const rootStyle: IStyle = {
   height: '100%',
   width: '100%',
-  gap: `${VERTICAL_GALLERY_GAP}rem`
+  gap: `${VERTICAL_GALLERY_GAP}rem`,
+  position: 'relative'
 };
 
 /**
  * @private
  */
 export const controlBarContainerStyle: IStyle = {
-  height: '2rem'
+  height: '2rem',
+  width: '100%',
+  position: 'absolute',
+  bottom: '0'
```

Comment:
Just to clarify this is to force the control bar to the bottom of the VerticalGallery?

Reply:
yes this might change but there might be a situation where a tile is fully expanded to the max and there is not enough room for another child but there is a gap to the local tile. I imagine Alex P will want it to be glued to the bottom